assets/js/components/ResetButton.test.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * ResetButton tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { render, createTestRegistry, fireEvent, act } from '../../../tests/js/test-utils';
+import { STORE_NAME as CORE_SITE } from '../googlesitekit/datastore/site/constants';
+import ResetButton from './ResetButton';
+import { subscribeUntil } from '../../../tests/js/utils';
+
+describe( 'ResetButton', () => {
+	let registry;
+	const adminURL = 'http://example.com/';
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		registry.dispatch( CORE_SITE ).receiveSiteInfo( { adminURL } );
```

Comment:
Please use `provideSiteInfo` from `tests/js/utils.js` instead using the default values.